plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2429,9 +2429,7 @@ public void testSplitPruningForFilterOnPartitionColumn()
         verifySplitCount("SELECT * FROM " + tableName + " WHERE regionkey < 2", 2);
         verifySplitCount("SELECT * FROM " + tableName + " WHERE regionkey < 0", 0);
         verifySplitCount("SELECT * FROM " + tableName + " WHERE regionkey > 1 AND regionkey < 4", 2);
-
-        // TODO(https://github.com/trinodb/trino/issues/9309) should be 1
```

Comment:
Mark your PR as fixing this issue (and https://github.com/trinodb/trino/issues/7608 too)

Reply:
How do I do that? I already have https://github.com/trinodb/trino/issues/9309 in the description

Reply:
add "fixes #nnnn" to PR description. then take a look at this section on the sidebar:

![image](https://user-images.githubusercontent.com/144328/140907459-ab72f59b-556e-4472-b089-2c8c18bfab7d.png)